Base/Python/slicer/util.py

```diff
@@ -3590,7 +3590,7 @@ def longPath(path):
     sysInfo = qt.QSysInfo()
     if sysInfo.productType() != 'windows':
         return path
-    # Skip prefixing relative paths as UNC prefix wors only on absolute paths
+    # Skip prefixing relative paths as UNC prefix was only on absolute paths
```

Comment:
Should be "works"

Reply:
Fixed in fac873f85ec03aa6fad367907c836ca3c2d74948